docs/user/masternodes/setup-evonode.rst

```diff
@@ -628,6 +628,18 @@ There are two ways to get SSL certificates:
    certificate files to your server. `SSL Dragon <https://www.ssldragon.com/>`_ and `SuperbitHost
    <https://www.superbithost.com/ssl-certificates/>`_ are two options that accept cryptocurrency.
 
+   Please make sure that a certificate chain file contains your server certificate at the top and
+   intermediate/root certificates if present. Usually, you need to concatenate the certificate file and
+   bundle if present.
```

Comment:
Re: "usually you need to concatenate ..."
Do you think we should show an example of how to do this? I'm not very familiar with this cert stuff, so I don't know if most people will understand or not.

Reply:
Not sure what example you want to give? How do we concatenate two files in case if user receives it separately?

Reply:
If you think it is clear as-is, we can go with it and update later if we get questions.

Reply:
If it's not clear for you then we probably need to give more information. What exactly is not clear for you?

Reply:
Oh, do they just need to do smth like `cat file1 file2 > combined_file` to actually concatenate the files. Or is there a specific ssl / cert related command?

Reply:
Yes, just like that: `cat certificate.crt bundle.crt > full_chain.crt`